<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="web-vk">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="headerBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="buttonOpenUserPage">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Открыть страницу</string>
         </property>
         <property name="icon">
          <iconset theme="cs-user">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editUserName">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonOpenUserPageGo">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="icon">
          <iconset theme="go-next">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonOpenLink">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Открыть в браузере</string>
         </property>
         <property name="icon">
          <iconset theme="applications-internet">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonUpdate">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Обновить</string>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonSettings">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Настройки</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="icon">
          <iconset theme="configure"/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonUser">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Имя</string>
         </property>
         <property name="icon">
          <iconset resource="res/resource.qrc">
           <normaloff>:/icons/icon.svg</normaloff>:/icons/icon.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonSwitchUser">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Сменить позьзователя</string>
         </property>
         <property name="text">
          <string>Смена пользователя</string>
         </property>
         <property name="icon">
          <iconset theme="system-reboot">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonLogout">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Завершить сеанс</string>
         </property>
         <property name="text">
          <string>Выход</string>
         </property>
         <property name="icon">
          <iconset theme="system-log-out">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonLogout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>logout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSwitchUser</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchSession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editUserName</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonOpenUserPageGo</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>switchSession()</slot>
  <slot>logout()</slot>
 </slots>
</ui>
